apollo-portal/src/main/resources/static/index.html

```diff
@@ -36,37 +36,221 @@
+                                        ng-click="toTop(app.favoriteId);$event.stopPropagation();">
+                                    <img src="img/top.png" class="i-15">
+                                </button>
+                                <button class="btn btn-default btn-xs" title="{{'Index.FavoriteCancel' | translate }}"
+                                        ng-click="deleteFavorite(app.favoriteId);$event.stopPropagation();">
+                                    <img src="img/like.png" class="i-15">
+                                </button>
+                            </td>
+                        </tr>
+                    </table>
+                </div>
+                <div class="no-favorites text-center" ng-show="!favorites || favorites.length == 0">
+                    <h4>{{'Index.FavoriteTip' | translate }}</h4>
+                </div>
+                <div ng-show="!hasMoreCreatedApps" style="height: 15px"></div>
+                <div style="width: 180px;margin-left: 30px;margin-top: 20px" ng-show="hasMoreFavorites"
+                     ng-click="getUserFavorites()">
+                    <div href="#" class="thumbnail hover cursor-pointer"
+                         style="display: flex;justify-content: center;align-items: center">
+                        <div><img class="more-img" src="img/more.png" /></div>
+                        <div style="margin-left: 5px"><h5>{{'Index.LoadMore' | translate }}</h5></div>
+                    </div>
+                </div>
+            </div>
+            <div class="public-namespace-list" ng-show="whichContent==='3'">
+                <div class="display-table" ng-show="publicNamespaces.length > 0">
+                    <table>
+                        <tr>
+                            <th style="width: 20%">{{'Index.appTable.appId' | translate }}</th>
+                            <th style="width: 30%">{{'Index.appTable.nameSpace' | translate }}</th>
+                            <th style="width: 20%">{{'Index.appTable.CreatedTime' | translate }}</th>
+                            <th style="width: 30%">{{'Index.appTable.LastModifiedTime' | translate }}</th>
+                        </tr>
+                        <tr ng-repeat="app in publicNamespaces" ng-click="goToAppHomePage(app.appId)"
+                            href="#" class="hover cursor-pointer">
+                            <td style="width: 20%">{{ app.appId }}</td>
+                            <td style="width: 30%">{{ app.name }}</td>
+                            <td style="width: 20%">{{ app.dataChangeCreatedTime.slice(0,10) }}</td>
+                            <td style="width: 30%">{{ app.dataChangeLastModifiedTime.slice(0,10) }}</td>
+                        </tr>
+                    </table>
+                </div>
+                <div class="no-public text-center" ng-show="!publicNamespaces || publicNamespaces.length == 0">
+                    <h4>{{'Index.FavoriteTip' | translate }}</h4>
```

Comment:
This looks like wrong?
```suggestion
                    <h4>{{'Index.PublicNamespaceTip' | translate }}</h4>
```